### DISCUSSION TITLE
Shipping method not saving correctly

### DISCUSSION THREAD

**Discussion by sstottelaar** [duncanmcclean/simple-commerce#1129] [Help]
Hi,

When opening an order in the back office with a shipping method, it's not properly displayed:

<img width="423" alt="image" src="https://github.com/user-attachments/assets/aeb48114-49f3-4267-99a0-fcffee10030d">

This is my shipping method:

```php

class ParcelShipping extends BaseShippingMethod implements ShippingMethod
{
    public function name(): string
    {
        return __('Pakketpost');
    }

    public function description(): string
    {
        return __('Wij sturen het pakket op naar je verzendadres.');
    }

    public function calculateCost(Order $order): int
    {
        return 795;
    }

    public function checkAvailability(Order $order, Address $address): bool
    {
        return true;
    }
}
```

Is there anything I'm doing wrong here? When I type to save the order in the backoffice I also get a validation error:

<img width="417" alt="image" src="https://github.com/user-attachments/assets/cb197a07-b604-423c-a921-1012288b1dd2">

Thanks in advance!

**Comment by duncanmcclean** [comment]
This seems like a bug. Are you able to [open a bug report](https://github.com/duncanmcclean/simple-commerce/issues/new/choose) for this instead?

**Reply by sstottelaar** [reply]
Will do!
